common/src/lib.rs

```diff
@@ -1779,19 +1785,19 @@ property_methods! {
     (TextSelection, text_selection, get_text_selection_property, Option<&TextSelection>, set_text_selection, set_text_selection_property, impl Into<Box<TextSelection>>, clear_text_selection)
 }
 
-impl Node {
+impl FrozenNode {
     option_properties_debug_method! { debug_option_properties, [transform, bounds, text_selection,] }
 }
 
-impl NodeBuilder {
+impl Node {
     option_properties_debug_method! { debug_option_properties, [transform, bounds, text_selection,] }
 }
 
 vec_property_methods! {
     (CustomActions, CustomAction, custom_actions, get_custom_action_vec, set_custom_actions, set_custom_action_vec, push_custom_action, push_to_custom_action_vec, clear_custom_actions)
 }
 
-impl fmt::Debug for Node {
+impl fmt::Debug for FrozenNode {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let mut fmt = f.debug_struct("Node");
```

Comment:
Did you not change the name of the struct on purpose? I think we still want the debug representation to show if it's a frozen node or not, right?